app/src/main/java/com/tvtracker/dto/User.kt

```diff
@@ -1,3 +1,4 @@
 package com.tvtracker.dto
 
-data class User(var uid: String = "", var displayName: String?)
+data class User(var uid: String = "",
```

Comment:
Classes with longer headers should be formatted so that each primary constructor parameter is in a separate line with indentation.
https://kotlinlang.org/docs/coding-conventions.html#class-headers